dipy/reconst/forecast.py

```diff
@@ -479,7 +480,7 @@ def lb_forecast(sh_order):
     diag_lb = np.zeros(n_c)
     counter = 0
     for l in range(0, sh_order + 1, 2):
-        for m in range(-l, l + 1):
+        for _ in range(-l, l + 1):
```

Comment:
Why not use the following? 

```suggestion
        for l in range(0, sh_order + 1, 2):
            diag_lb[l//2] = (l * (l + 1)) ** 2
```
And get rid of the counter variable? Wouldn't that have the same effect?

Reply:
I do not think so, because in your case, we will not fill `diag_lb`, it misses some iteration or another trick